web/src/constants/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const VEHICLE_TAGS: string[] = [
+  "extraordinary damage",
+  "ordinary wear and tear",
+  "consumable",
+] as const
+
```

Comment:
pabutang ko ani na comment
```suggestion
// Reference to `category` table, primary key id = 6 (Vehicle Repair & Maintenance)
```